.github/workflows/functional-arm64.yaml

```diff
@@ -0,0 +1,30 @@
+name: functional-arm64-tests
+on: [push, pull_request]
+permissions: read-all
+jobs:
+  test:
+    runs-on: [Linux, ARM64]
```

Comment:
This seems not correct to me. `Linux` is `GOOS` type, while `ARM64` is `GOARCH` type.

Reply:
@ahrtr these are tags you see on the GH self hosted runner.

![image](https://user-images.githubusercontent.com/23304/214157509-3df226a0-f984-49dc-b841-bb2e4b781665.png)

Reply:
Got it, it's a combination of the tags. 

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idruns-on

Reply:
thx @dims , please see my another comment below.

Reply:
yep done :)